instrumentation/net/http/httptrace/otelhttptrace/clienttrace_test.go

```diff
@@ -5,9 +5,20 @@ package otelhttptrace
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"net/http"
 	"net/http/httptrace"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
```

Comment:
This means the sdk is being added as a dependency, which instrumentations shouldn't have (someone using this instrumentation may want to use their own custom SDK).

Could you move those tests into the `test` subfolder?

Reply:
I have removed these, and added `TestEndBeforeStartWithoutSubSpansDoesNotPanic` instead in `test` subfolder.